README.md

```diff
@@ -86,7 +86,150 @@ There are several ways to execute the code cells in your notebook:
     panel. Here you can schedule your notebook to be executed once at some future
     time, or repeatedly at your specified interval.
 
-## 4. Save and Share
+## 4. Analyze the Results
+
+After running each cell of the notebook, the results will display. When we use pixiedust display() to create an interactive dataset, we are able to manipulate the data using tables, graphs, and charts.
+
+### Options for PixieDust Charts
+
+After running cell #3 "display(accidents)", we can see by clicking the "Options" button that we are able to manipulate the keys and values for the fields used in the chart:
+![](docs/sources/pixieChartOptions.png)
+
+Following the instructions, we use DaysOfWeek and IncidntNum, but the user can change the keys and value to see how the chart will look with different inputs.
+
+### Use Spark SQL to query data
+
+We use Spark SQL to isolate data to the Taraval district:
+```
+accidents.registerTempTable("accidents")
+taraval = sqlContext.sql("SELECT * FROM accidents WHERE PdDistrict='TARAVAL'")
+```
+We then get an interactive map of the Taraval district:
+![](docs/sources/taravalSQLresults.png)
+
+### Create a PixieApp Dashboard
+
+With PixieApps, we can create a dashboard with map layers that can be used to visualize various datasets (i.e. Speeding, Traffic Calming, Police Districts, and Crimes):
+![](docs/sources/pixieAppsLayers.png)
+
+#### Create the skeleton
+```
+from pixiedust.display.app import *
+
+@PixieApp
+class SFDashboard():
+    def mainScreen(self):
+        return """
+<div class="well">
+    <center><span style="font-size:x-large">Analyzing San Francisco Public Safety data with PixieDust</span></center>
+    <center><span style="font-size:large"><a href="https://datasf.org/opendata" target="new">https://datasf.org/opendata</a></span></center>
+</div>
+<div class="row">
+    <div class="form-group col-sm-2" style="padding-right:10px;">
+        <div><strong>Layers</strong></div>
+        {% for layer in this.layers %}
+        <div class="rendererOpt checkbox checkbox-primary">
+            <input type="checkbox" pd_refresh="map{{prefix}}" pd_script="self.toggleLayer({{loop.index0}})">
+            <label>{{layer["name"]}}</label>
+        </div>      
+        {%endfor%}
+    </div>
+    <div class="form-group col-sm-10">
+        <div id="map{{prefix}}" pd_entity pd_options="{{this.formatOptions(this.mapJSONOptions)}}"/>
+    </div>
+</div>
+"""
+```
+#### Create the Map of Accidents
+```
+<div id="map{{prefix}}" pd_entity pd_options="{{this.formatOptions(this.mapJSONOptions)}}"/>
+```
+pd_entity: tell PixieDust which dataset to work on 
```

Comment:
capitalize Tell
not sure if I'd put periods on these but I guess I have been for full sentences. The 'Contains' one is just a fragment but I think its Cap/period is supposed to just follow what you do on this one.